### PR TITLE
 Add test kitchen with test policy, dummy tests and fix version in metadata

### PR DESCRIPTION
**Pull request by zirkov**
* Added .kitchen.yml, test/recipes and test/policyfiles
* Fixed version in metadata.rb to match CHANGELOG and github versions

### CODE REVIEW

**Comment by Brantone** [comment]
Should `test/policyfiles/teamcity_node.rb` be in this cookbook?

File: metadata.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures Visual Studio'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.0.0'
+version          '1.2.6'
```
Comment (Brantone) [inline metadata.rb:11]: Did this fall back a version?

**Comment by zirkov** [comment]
@Brantone actually, I think we're good enough to drop the extra kitchen steps completely (and test them inside the teamcity cookbook anyway); will update the PR to include only generic parts

File: metadata.rb (hunk 1 above)
Comment (zirkov) [reply metadata.rb:11]: That was actually me accidentally copying full metadata from upstream without noticing the major version bump
It was never intended to be on v2 in this fork anyway

**Pull request merged**